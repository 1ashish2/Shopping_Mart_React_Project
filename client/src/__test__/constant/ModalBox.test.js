

import { render, cleanup } from "@testing-library/react";
import { MemoryRouter } from 'react-router-dom'
import ModalBox from "../../constant/ModalBox.constant";



afterEach(() => {
    cleanup();
})

describe("Modalbox container Component", () => {
   
    render(<ModalBox  />, { wrapper: MemoryRouter });
    it("matches snapshot", () => {
        const { asFragment } =  render(<ModalBox  />, { wrapper: MemoryRouter });

        expect(asFragment()).toMatchSnapshot();
    })
    
    
    
   
})