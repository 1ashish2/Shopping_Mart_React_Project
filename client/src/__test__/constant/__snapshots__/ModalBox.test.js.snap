// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modalbox container Component matches snapshot 1`] = `<DocumentFragment />`;
